README.md

```diff
@@ -298,6 +298,31 @@ the private key. This will be the passphrase used when configuring
 
 The `certificate.pem` file that is generated can be used with **grocer**.
 
+### Alternative way to generate certificate file (no macos required)
```

Comment:
At first I read `macos` as `macros`. Perhaps change it to the proper noun spelling of `Mac OS X`?